Content.Client/EscapeMenu/UI/Tabs/KeyRebindTab.xaml.cs

```diff
@@ -108,7 +108,7 @@ void AddCheckBox(string checkBoxName, bool currentState, Action<BaseButton.Butto
             AddHeader("ui-options-header-interaction-basic");
             AddButton(EngineKeyFunctions.Use);
             AddButton(ContentKeyFunctions.WideAttack);
-            AddButton(ContentKeyFunctions.ActivateItemInHand);
+            AddButton(ContentKeyFunctions.UseItemInHand);
             AddButton(ContentKeyFunctions.AltActivateItemInHand);
```

Comment:
will do. Ill also move the kybinding building from server to shared so they get predicted

Reply:
Also, because AFAIK it will break any custom key bindings people have set up, I haven't changed the string name.

i.e., its still
```
public static readonly BoundKeyFunction UseItemInHand = "ActivateItemInHand";
```
I'll probably just update than in a separate PR after this one with a message in a changelog. Might also rename `ActivateItemInWorld` to something else given there is no other activate keybinding. Unless we plan to actually add an Activate-in-hand keybinding,